source/articles/how-to/teach-linq2db-convert-custom-net-code-to-sql.md

```diff
@@ -46,40 +46,40 @@ Let's test it:
 [Test]
 public void SqlExpressionAttributeTest()
 {
-  using (var db = new DataModel())
-  {
-    db.InlineParameters = true; // inlined parameters can be helpful when debugging
-    db.Customers.Where(c => c.DateOfBirth.Between(new DateTime(2000, 1, 1), new DateTime(2000, 12, 31))).ToList();
-  }
+    using (var db = new DataModel())
+    {
+        db.InlineParameters = true; // inlined parameters can be helpful when debugging
+        db.Customers.Where(c => c.DateOfBirth.Between(new DateTime(2000, 1, 1), new DateTime(2000, 12, 31))).ToList();
+    }
 }
 ```
 
 The SQL generated for SQL Server 2012 is:
 
 ```sql
 SELECT
- [t1].[ID],
- [t1].[DateOfBirth],
- [t1].[FirstName],
- [t1].[LastName],
- [t1].[Email]
+    [t1].[ID],
+    [t1].[DateOfBirth],
+    [t1].[FirstName],
+    [t1].[LastName],
+    [t1].[Email]
 FROM
- [dbo].[Customer] [t1]
+    [dbo].[Customer] [t1]
 WHERE
- [t1].[DateOfBirth] BETWEEN '2000-01-01' AND '2000-12-31'
+    [t1].[DateOfBirth] BETWEEN '2000-01-01' AND '2000-12-31'
  ```
 
 Notice the use of the `Sql.ExpressionAttribute.PreferServerSide` property set to true. `PreferServerSide = true` tells LINQ to DB to convert the method to SQL if possible, and if it's not possible for some reason - then to execute the method locally.
 
-There is another similar property � `ServerSideOnly`. If it's set to True, LINQ to DB will throw an exception if it can't convert a method to SQL. It can be set to true when you can't, don't need or don't want to write a client-side implementation.
+There is another similar property � `ServerSideOnly`. If it's set to True, LINQ to DB will throw an exception if it can't convert a method to SQL. It can be set to true when you can't, don't need or don't want to write a client-side implementation.
```

Comment:
```suggestion
There is another similar property - `ServerSideOnly`. If it's set to True, LINQ to DB will throw an exception if it can't convert a method to SQL. It can be set to true when you can't, don't need or don't want to write a client-side implementation.
```